src_code/src/utils/services/helper_functions/db_helper_functions/DBHelperFunctions.js

```diff
@@ -0,0 +1,47 @@
+async function GetUserInfoBasedOnEmailAddress(db, emailAddress){ 
+    try {
+        /** Initailize an user object query from database */
+        const user = await db.select(
+            "PK_USER_ID",
+            "FK_CDE_ROLE", 
```

Comment:
As we discussed in the database schema, we will not using the user_role table anymore, so instead of "FK_CDE_ROLE", we can just named it "ROLE" instead.